src/plugins/CesiumUsdSchemas/src/CesiumUsdSchemas/tileset.h

```diff
@@ -24,10 +24,10 @@ PXR_NAMESPACE_OPEN_SCOPE
 class SdfAssetPath;
 
 // -------------------------------------------------------------------------- //
-// CESIUMTILESETSCHEMAAPI                                                     //
+// CESIUMTILESETPRIM                                                          //
 // -------------------------------------------------------------------------- //
 
-/// \class CesiumTilesetAPI
+/// \class CesiumTileset
 ///
 /// Adds Cesium specific data to a prim for representing a tileset.
```

Comment:
I re-ran `regenerate_schema.bat` and all looked good, except this doc which needed to be updated.

```suggestion
/// A prim representing a tileset.
```